src/cli/import.rs

```diff
@@ -0,0 +1,156 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+use pixi_config::{Config, ConfigCli};
+use pixi_manifest::{DependencyOverwriteBehavior, FeatureName, PrioritizedChannel, SpecType};
+use pixi_spec::PixiSpec;
+use pixi_utils::conda_environment_file::CondaEnvFile;
+use rattler_conda_types::Platform;
+
+use miette::IntoDiagnostic;
+
+use super::cli_config::LockFileUpdateConfig;
+use crate::{
+    WorkspaceLocator,
+    cli::cli_config::{PrefixUpdateConfig, WorkspaceConfig},
+    environment::sanity_check_workspace,
+};
+
+#[derive(Parser, Debug, Default)]
+#[clap(arg_required_else_help = true, verbatim_doc_comment)]
+pub struct Args {
+    #[clap(flatten)]
+    pub workspace_config: WorkspaceConfig,
+
+    /// File to import into the workspace.
+    #[arg(required = true, id = "FILE")]
+    pub file: PathBuf,
+
+    // File is to be interpreted as specifying a conda environment.
+    #[arg(long)] // conflicts_with = "pypi"
+    pub conda: bool,
+
+    /// The platforms for the imported environment
+    #[arg(long = "platform", short, value_name = "PLATFORM")]
+    pub platforms: Vec<Platform>,
+
+    /// A name for the created feature and environment
+    #[clap(long, short, value_name = "ENVIRONMENT")]
+    pub environment: Option<String>,
+
+    /// A name for the created feature and environment
+    #[clap(long, short, value_name = "FEATURE", conflicts_with = "environment")]
+    pub feature: Option<String>,
```

Comment:
So, the two flags do the same thing?

How about, if you specify `--feature`, the environment will have the same name.
And if you specify `--environnment`, the feature will have the same name.

But if you can still specify both flags